how2meet/main.py

```diff
@@ -2,127 +2,36 @@
 """
 Main entrypoint
 """
+import os, logging
+
 from fastapi import Depends, FastAPI, HTTPException
 from nicegui import ui
 from sqlalchemy.orm import Session
 
 from .db import crud, models, schemas
 from .db.database import engine, get_db
 from .ui import frontend
+from .routers import events, invites
```

Comment:
I read "events" and "invites" as the entities themselves not their routers... could we name them something else or just import each router directly rather than from the parent folder?